content/blog/webrtc-demo.md

```diff
@@ -0,0 +1,192 @@
+Title: Creating our WebRTC demo
+Date: 2021-10-29 16:30:00
+Author: Manon Gerray
+Category: Wazo Platform
+Tags: wazo platform, webrtc, front-end
+Slug: creating-our-webrtc-demo
+Status: published
+
+## Introduction
+
+Hello everyone! Six months ago I started a one year work/study training program at Wazo as a front-end developer. Wazo has a huge development framework to get used to, and what is better than a concrete project to learn the Wazo SDK? 
+
+That is how I have been assigned the Web RTC demonstration project, which will be useful to anyone wondering what we are able to do with the Wazo SDK. The main goals I had when I took over the project were the following : 
+* Totally redesign the application to improve user experience
+* Set up phone & video call, transfer & indirect transfer
+* Being able to call another number while already on call & switch between interlocutors
+
+## Launching the demo at home
+
+If you want to take a closer look at the code and the demo itself, you can go and set it up with these commands: 
+```
+apt install wazo-plugin-cli
+wazo-plugind-cli -c 'install git https://github.com/wazo-platform/wazo-webrtc-demo'
+```
+Otherwise, you can clone the following repository : [wazo-webrtc-demo](https://github.com/wazo-platform/wazo-webrtc-demo) and follow the instructions in the README file on the master branch. 
+
+Feel free to report any bugs, I will occasionally take a look.  
+
+
+## Redesigning the interface
+
+I prepared some screenshots so you can notice the difference between the application before and after I redesigned it. The main styling was based on a few mock ups. You can take a look here at the home page mock up. 
+
+![Mockup](/images/blog/webrtc-demo/mockup.png "Mock up")
+
+What you can’t see in the screenshots is the way the interface is moving. I decided to animate every button : when you hover them, they fill up. The same goes for the logout button : when you hover it, it spins backwards and the real logout button appears. I also looked up on the internet to find better backgrounds : a landscape at dawn for the login page, and a landscape at daylight for the rest of the application. These images are royalty-free and created by Jcomp (source: Freepik).
```

Comment:
Maybe it will be better to move the copyright thing to the end of the text to avoid to break reading
I guess @sduthil will have lot of suggestion on How to :smile:

Reply:
+1! My suggestion is to add the source note in a small paragraph just after the images, I did not find a better way.